core/common/src/test/scala/net/liftweb/common/HListSpec.scala

```diff
@@ -81,6 +81,8 @@ object HListSpec extends Specification {
 
           success
         }
+
+        case Right(_) => failure
```

Comment:
This just fixed an exhaustive match warning during lift-common spec compilation.